app/index.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
 const _         = require('lodash');
-const yosay     = require('yosay');
-const superb    = require('superb');
-const mkdirp    = require('mkdirp');
 const chalk     = require('chalk');
 const Generator = require('yeoman-generator');
+const mkdirp    = require('mkdirp');
+const superb    = require('superb');
+const yosay     = require('yosay');
+
+const OPTIONS = {
+    KEEP_SILENCE: 'keep-silence',
```

Comment:
Not sure whether you use such a naming style for defining a constants or not... It also supposed to behave a bit like a enum, so it gets complicated. Let me know if we need it at all and if you have any thouhts regarding naming.

Reply:
I like the way, in this way they are easy to identify :)